tests/regression/tests/REQUEST-942-APPLICATION-ATTACK-SQLI/942310.yaml

```
@@ -1,23 +1,57 @@
 ---
-  meta:
-    author: "Christian S.J. Peron"
-    description: None
-    enabled: true
-    name: 942310.yaml
-  tests:
-  -
-    test_title: 942310-1
-    desc: "chained SQL injection attempts 2/2"
+meta:
+  author: "Christian S.J. Peron"
+  description: None
+  enabled: true
+  name: 942310.yaml
+tests:
+  - test_title: 942310-1
+    desc: |
+      Chained SQL injection attempts 2/2.
+      This test originally checked the expression `[\"'`]\s+and\s*?=\W`. We opted to remove that expression as it
+      does not appear to match anything useful (https://github.com/coreruleset/coreruleset/issues/2118).
+      This test now checks that no match occurs for something that would have matched the original expression.
     stages:
-    -
-      stage:
-        input:
-          dest_addr: 127.0.0.1
-          headers:
-            Host: localhost
-          method: GET
-          port: 80
-          uri: "/?var=%22%27%20and%20%3d%20bar"
-          version: HTTP/1.0
-        output:
-          log_contains: id "942310"
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+            method: GET
+            port: 80
+            uri: "/?var=%22%27%20and%20%3d%20bar"
+            version: HTTP/1.0
+          output:
+            no_log_contains: id "942310"
+  - test_title: 942310-2
+    desc: |
+      Chained SQL injection attempts 2/2.
+      This is a regular test, this is a valid payload: \(\s*?select\s*?\w+\s*?\(
```

Pleas change this to:

```suggestion
This test checks for a positive match of `\(\s*?select\s*?\w+\s*?\(`.
```